packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { instantSearchContext } from 'react-instantsearch-core';
+import { createConcurrentSafePromise } from '../lib/createConcurrentSafePromise';
+import { debounce } from '../lib/debounce';
+
+/* eslint-disable react/prop-types */
+function DefaultAnswersComponent({ isLoading, hits = [] }) {
+  return isLoading ? (
+    <p>loading...</p>
+  ) : (
+    hits && hits.length > 0 && <pre>{JSON.stringify(hits, null, 2)}</pre>
+  );
+}
+
+export default function Answers({
+  searchClient,
+  queryLanguages,
+  attributesForPrediction = ['*'],
```

Comment:
```suggestion
  attributesForPrediction,
```

Reply:
let the engine set this default

Reply:
https://github.com/algolia/react-instantsearch/pull/2996/commits/194cddb48f2fb9b9a789a185e5efc4610c65759f